### ISSUE TITLE
about loss functions in ssd training

### ISSUE THREAD

**Issue by yuconglin** [zhreshold/mxnet-ssd#87]
Hi @zhreshold zhreshold:
    In train_net.py, I observed that a pretrained vgg model is loaded and modified, but I didn't find that a new loss function (the loss function as in the paper) is added in the model, and in this case, the original Softmax loss function will be used.
    Can I know in which file you added the new loss function to the vgg model before training?

THanks a lot.

**Comment by yuconglin** [comment]
I figured out, sorry
